XRTK-Core/Packages/com.xrtk.core/Interfaces/InputSystem/Controllers/Hands/ISimulatedControllerDataProvider.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+using XRTK.Interfaces.Providers.Controllers;
+using XRTK.Providers.Controllers;
+
+namespace XRTK.Interfaces.InputSystem.Controllers.Hands
+{
+    public interface ISimulatedControllerDataProvider : IMixedRealityControllerDataProvider
+    {
+        /// <summary>
+        /// The simulated update frequency in milliseconds mimics the hardware's ability to
+        /// update controller tracking data. A value of 0ms will provide data
+        /// updates every frame.
+        /// </summary>
+        double SimulatedUpdateFrequency { get; }
+
+        /// <summary>
+        /// Time after which uncontrolled controllers are hidden
+        /// </summary>
+        float ControllerHideTimeout { get; }
+
+        /// <summary>
+        /// Default distance of the controller from the camera.
+        /// </summary>
+        float DefaultDistance { get; }
+
+        /// <summary>
+        /// Depth change when scrolling the mouse wheel.
+        /// </summary>
+        float DepthMultiplier { get; }
+
+        /// <summary>
+        /// Apply random offset to the controller position.
+        /// </summary>
+        float JitterAmount { get; }
+
+        /// <summary>
+        /// Key to toggle persistent mode for the left controller.
+        /// </summary>
+        KeyCode ToggleLeftPersistentKey { get; }
```

Comment:
I'd like to update these to `MixedRealityInputActions` at some point in the future.

Reply:
Agreed. Would love to see that as well in a not so distant future update.